neo4j/config/driver.go

```diff
@@ -48,6 +39,18 @@ type Config struct {
 	// The InsecureSkipVerify attribute of TlsConfig is always derived from the initial URI scheme.
 	// The ServerName attribute of TlsConfig is always derived from the initial URI host.
 	//
+	// Use case: Trust self-signed certificates:
```

Comment:
maybe move this to a documented example (in `driver_example_test.go` iirc)

Reply:
which makes me think... shouldn't the `driver_with_context_{test,testkit,example_test}.go` files be renamed to remove the `_with_context` bit?